src/pages/Product/jeein/Product.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import './Product.scss';
+
+function Product() {
+  const params = useParams();
+
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/sample.json')
+      .then(response => response.json())
+      .then(result => setUsers(result));
+  }, []);
```

Comment:
users..?? 👀👀

변수명은 products 를 다루는 것들이라 이렇게 바꾸는게 더 좋을 것 같습니다.
JSX 구문도 바뀐 변수명으로 수정하여주세요
해당 변수명에 F2 를 누르면 한번에 연결되어있는 변수들을 바꿀 수 있답니다!! (🍯팁)

```suggestion
  const [products, setProducts] = useState([]);

  useEffect(() => {
    fetch('http://localhost:3000/data/sample.json')
      .then(response => response.json())
      .then(setProducts);  // callback으로 들어오는 parameter와 argument가 같다면 왼쪽 코드처럼 생략도 가능해요!
  }, []);
```